app/components/earthworks/constraints_component.rb

```diff
@@ -0,0 +1,9 @@
+# Override allows for:
+# 1. moving of start button to end of selected items
+# 2. constraints-label not being visually-hidden
+# 3. To override default query_constraint_component, facet_constraint_component
+# so we can get the HTML in for creating pills from the component library
+module Earthworks
```

Comment:
Can you remove the Earthworks module from this component?

Reply:
done